app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/startregistration/StartRegistrationScreen.kt

```diff
@@ -264,18 +263,14 @@ private fun StartRegistrationContent(
                     .testTag("RegionSelectorDropdown"),
             )
             if (isNewOnboardingUiEnabled) {
-                IconButton(
+                BitwardenStandardIconButton(
+                    iconRes = R.drawable.ic_tooltip_small,
```

Comment:
Doesn't this sort of unsafely assume we'll be using a vector resource without actually enforcing that? Wouldn't using a painter be safer? Or at least calling this "vectorIconRes" maybe?